elements/src/components/pos-dialog/pos-dialog.css

```diff
@@ -0,0 +1,57 @@
+:host {
+  font-family: var(--font-sans);
+  display: block;
+}
+
+button#new {
```

Comment:
you do not need the `button#new` style here, this was for the +-Button in `pos-add-new-thing` which is not part of this component